README.md

```diff
@@ -9,7 +9,7 @@ List of Cocoa conferences for iOS & macOS developers.
 
 ## Adding a new conference
 
-The site is generated from [this file](https://github.com/lascorbe/CocoaConferences/_data/conferences.yml); add new conferences there in YAML format.
+The site is generated from [this file](https://github.com/Lascorbe/CocoaConferences/blob/master/_data/conferences.yml); add new conferences there in YAML format.
```

Comment:
You can also just change the link to `_data/conferences.yml` so its relevant on every branch.. not a big deal